zynqmp.mk

```diff
@@ -1,8 +1,8 @@
 ROOT			= $(PWD)/..
 PLATFORM		?= zcu102
-PETALINUX_PATH		?= /opt/Xilinx/petalinx_2018_2
+PETALINUX_PATH		?= $(PETALINUX)
```

Comment:
I cannot see that `$(PETALINUX)` is defined elsewhere? You want to enforce this to be provided from command line? Isn't it possible to have some common ground to start from as a default?

Reply:
Is this in the `settings.sh` file? That comes with the petalinux package?

Reply:
Yes `$(PETALINUX)` is defined by when you source `settings.sh`